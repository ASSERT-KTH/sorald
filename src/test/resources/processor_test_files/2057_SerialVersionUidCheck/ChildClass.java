package processor_test_files.2057_SerialVersionUidCheck;

public class ChildClass extends UpperClass {
  
}
