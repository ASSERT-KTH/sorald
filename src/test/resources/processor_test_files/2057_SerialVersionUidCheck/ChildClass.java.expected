package processor_test_files.2057_SerialVersionUidCheck;

public class ChildClass extends UpperClass {
  
  private static final long serialVersionUID = 1L;
}
